l20n/migrate/__init__.py

```diff
@@ -0,0 +1,5 @@
+# coding=utf8
+
+from context import MergeContext
+from transforms import CONCAT, COPY, EXTERNAL, PLURALS, REPLACE, SOURCE
+from changesets import convert_blame_to_changesets
```

Comment:
Thanks for link.

Re. versioning, do we want to keep migrations runnable in the future, even if the transforms change?  For instance, adding more complex placeable replacements (as discussed elsewhere in this PR) might require changes to how to `REPLACE` transform works.  What should be our compatibility story?